src/command.ts

```diff
@@ -94,3 +116,46 @@ export class Command {
         })
     }
 }
+
+/**
+ * Whether the flatpak-builder exists on host
+ */
+export function flatpakBuilderHostExists(): boolean {
+    if (FLATPAK_BUILDER_HOST_EXISTS === undefined) {
+        const command = new Command('flatpak-builder', ['--version'])
+
+        try {
+            const version = command
+                .execSync()
+                .toString()
+                .replace('flatpak-builder', '')
+                .trim()
+            FLATPAK_BUILDER_HOST_EXISTS = true
+            console.log(`host flatpak-builder version: ${version}`)
+        } catch (error) {
+            FLATPAK_BUILDER_HOST_EXISTS = false
+            console.log(`host flatpak-builder not found: ${error as string}`)
+        }
+    }
+
+    return FLATPAK_BUILDER_HOST_EXISTS
+}
+
+/**
+ * Whether the flatpak-installed org.flatpak.Builder exists
+ */
+export function flatpakBuilderSandboxedExists(): boolean {
+    if (FLATPAK_BUILDER_SANDBOXED_EXISTS === undefined) {
+        const flatpakBuilderEntry = getAvailable('app').find(app => app.id === 'org.flatpak.Builder')
```

Comment:
checking if flatpak-builder in sandbox exist through flatpak run org.flatpak.Builder --version seems simpler

Reply:
Using getAvailable looks good to me, might be a bit slow if you have too many apps installed. Can't we make use of flatpak info here instead?

Reply:
How would that be implemented?